### PR TITLE
Add pagination

### LINKED ISSUE #1039: Rétablir la pagination sur la préviz

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Close https://github.com/etalab/data.gouv.fr/issues/1039 with https://github.com/etalab/udata-front/pull/200

Note : publish @etalab/udata-front-plugins-helper stable version before merging this PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   },
   "devDependencies": {
     "@etalab/explore.data.gouv.fr": "^1.0.1",
-    "@etalab/udata-front-plugins-helper": "^1.0.1",
+    "@etalab/udata-front-plugins-helper": "^1.0.2-7",
```
Comment (maudetes) [inline package.json]: TODO: don't forget :)

File: udata-tabular-preview/explore.vue [hunk 2]
```diff
@@ -74,11 +83,40 @@ export default defineComponent({
...
+     *
+     * @param {import("@etalab/explore.data.gouv.fr/lib/csvapi").CsvapiResponse} res
+     */
+    const update = (res) => {
+      if (res.ok) {
+          rows.value = res.rows;
+          columns.value = res.columns;
+          rowCount.value = res.total;
+          columnCount.value = res.columns.length;
+          totalRows.value = res.total;
```
Comment (maudetes) [inline udata-tabular-preview/explore.vue]: Do we need to have both `rowCount` and `totalRows`?

**Review by maudetes — COMMENTED** [review]
:clap: :clap: :clap:

Now that we can paginate, I wonder if we should fix the row height, to prevent jump when changing page?
It doesn't have to be made in this PR if it's not trivial though

File: udata-tabular-preview/explore.vue (hunk 2 above)
Comment (nicolaskempf57) [reply udata-tabular-preview/explore.vue]: No, I'll remove `totalRows` to use the existing `rowCount`.

File: package.json (hunk 1 above)
Comment (nicolaskempf57) [reply package.json]: I'll leave this discussion open until we update it

**Comment by nicolaskempf57** [comment]
> wonder if we should fix the row height, to prevent jump when changing page?

We can't really do anything about it, because it depends of the row contents. This is the same for the column sizes.
We can add a max-height but this won't really solve the issue

**Review by maudetes — APPROVED** [review]
Seems all clear

File: package.json (hunk 1 above)
Comment (nicolaskempf57) [reply package.json]: v1.1.0 is live !

**Pull request merged**